website/docusaurus.config.ts

```diff
@@ -1,8 +1,8 @@
-import {themes as prismThemes} from 'prism-react-renderer';
-import type {Config} from '@docusaurus/types';
-import type {Options as IdealImageOptions} from '@docusaurus/plugin-ideal-image';
+import { themes as prismThemes } from 'prism-react-renderer';
```

Comment:
In most of the pages we don't use white spaces here, maybe we need to edit the .editorconfig or something in order to make sure autoformat doesn't add spaces here.

Reply:
adding this to the package.json would enforce this but I kind of prefer the spaces tbh :smile: 
```
  "prettier": {
    "bracketSpacing": false
  },
```

Reply:
I wouldn't say most, there are quite a few with spaces too (probably the ones I edited?).  I have no preference, but I would definitely make it consistent.